web/themes/custom/pineapple/sass/base/forms/_form-items.scss

```diff
@@ -44,7 +51,21 @@ form {
     }
   }
 
+  .form-item-date,
+  .form-item-password {
+    max-width: 33em;
+  }
+
+  .form-radio,
+  .form-checkbox {
+    margin-right: $unit;
+  }
+
   .fieldset-legend {
     font-size: 20px;
   }
+
+  .webform-button--submit.button {
+    background-color: white;
+  }
```

Comment:
What's this for?

Reply:
on mobile, the submit button has a gray background

Reply:
Why only on mobile?

Reply:
I have no clue what causes it.

Reply:
You'll have to show me, what's going on.